src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -1,13 +1,12 @@
 package mate.jdbc.dao;
 
 import java.util.List;
-import java.util.Optional;
 import mate.jdbc.model.Manufacturer;
 
 public interface ManufacturerDao {
     Manufacturer create(Manufacturer manufacturer);
 
-    Optional<Manufacturer> get(Long id);
+    Manufacturer get(Long id);
```

Comment:
usually we need to return Optional from dao layer and entity from service layer, let's return it as it was

Reply:
ok, I fixed the code to return Optional from DriverDao & ManufacturerDao and entities from DriverService & ManufacturerService